libcontainerd/container.go

```diff
@@ -0,0 +1,181 @@
+package libcontainerd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/Sirupsen/logrus"
+	containerd "github.com/docker/containerd/api/grpc/types"
+	"golang.org/x/net/context"
+)
+
+const (
+	configFilename = "config.json"
+	initProcessID  = "init"
+)
+
+type container struct {
+	process
+	pauseMonitor
+	restartManager RestartManager
+	restarting     bool
+	oom            bool
+	processes      map[string]*process
+}
+
+// RestartManager controls automatic restarting if container exits.
+type RestartManager interface {
+	ShouldRestart(exitCode uint32) (bool, chan error, error)
+	Cancel() error
+}
```

Comment:
The same interface is defined in the `restartmanager` package. Could one package import the other to avoid the duplicate definition?